src/QMCWaveFunctions/OrbitalBase.h

```diff
@@ -44,7 +44,7 @@ struct NLjob
     walker(w), elec(e), numQuadPoints(n)
   { }
 };
-#endif
+#endif // defined(QMC_CUDA) && !defined(ENABLE_SOA)
```

Comment:
Your #endif says it's for `#if defined(QMC_CUDA) && !defined(ENABLE_SOA)` while the the code reads `if defined(QMC_CUDA)`. Which one is correct?

Reply:
The first, I had a more ambitious port going of which this is a cut down PR. Will fix.